specification/resources/load_balancers/examples/go/create_load_balancer.yml

```
@@ -15,7 +15,7 @@ source: |-
 
       createRequest := &godo.LoadBalancerRequest{
           Name:      "example-01",
-          Size:      "lb-medium",
+          Size_unit: "1",
```

```suggestion
          SizeUnit: "1",
```

This is actually `SizeUnit` in godo. In general, Go code uses Pascal Case 

https://github.com/digitalocean/godo/blob/4b7fcd34ac65b0b5e05e8c074f3263b9e99b01c5/load_balancers.go#L37